src/emails/emails.ts

```diff
@@ -47,4 +49,11 @@ export class Emails {
     );
     return data;
   }
+
+  async cancelSchedule(id: string): Promise<CancelScheduleResponse> {
```

Comment:
the usage should be `resend.email.schedule.remove(id)` instead of `resend.emails.cancelSchedule(id)`

You probably can add this to the class

```
readonly schedule: Schedule;

constructor(private readonly resend: Resend) {
    this.schedule = new Schedule(resend);
}
```

and then define the `Schedule` class similar to the `email`